frontend/src/metabase/dashboard/actions/data-fetching-typed.ts

```diff
@@ -137,10 +144,10 @@ export const fetchDashboard = createAsyncThunk(
       }
 
       if (result.param_values) {
-        dispatch(addParamValues(result.param_values));
+        await dispatch(addParamValues(result.param_values));
```

Comment:
why should we await here?

Reply:
To avoid metadata updating in the next tick after results are displayed on the screen

Reply:
sounds weird, maybe we need to make rendering faster instead?